docs/modules.md

```diff
@@ -0,0 +1,94 @@
+[@fluencelabs/fluence](README.md) / Exports
```

Comment:
It would be nice to have a few short descriptions of the high-level classes and variables declared here. Like `FluencePeer`, `KeyPair`, etc

Reply:
That's not needed. The index of exports is like 5 cm to the right of the main content